src/Sulu/Bundle/WebsiteBundle/Admin/WebsiteAdmin.php

```diff
@@ -44,4 +49,21 @@ public function getJsBundleName()
     {
         return 'suluwebsite';
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getSecurityContexts()
+    {
+        return [
+            'Sulu' => [
+                'Website' => [
+                    'sulu.website.cache' => [
```

Comment:
Should we really introduce a new permission for that? Apart from that, the button always clears the cache for all the webspaces, right? Otherwise it might be a good solution to check the `PUBLISH` permission for a certain webspace. /cc @chirimoya